### PR TITLE
website completed

### PR DESCRIPTION
**Pull request by AntKarpo**

### CODE REVIEW

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 1]
```diff
@@ -30,19 +30,20 @@ the start of the introduction. */
   --regular-text: 1rem;
   --regular-line-height: 1.5rem;
 
-  /* Calculations */
+  /* Calculations    16px=1rem   */
 
-  /* Calculating the header height */
+  /* Calculating the header height 
+          header-height = 12.6rem 202px */
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css]: In this context it's fine to leave but in a professional context you probably wouldn't want to leave these types of comments in there, just fyi :)

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 2]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
 /* Media Query Tablet */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
-
-/* Media Query Desktop */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
+@media screen and (min-width: 600px) {
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css]: where does this value of `600px` come from?

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 3]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
+  .card-section {
+    display: flex;
+    justify-content: center;
+    flex-flow: wrap;
+    gap: var(--gap);
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css:251]: CSS variable, nice!

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 4]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
+  }
+
+  .product-card {
+    flex-wrap: wrap;
+    width: 40%;
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css]: Why `40%`?

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 5]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
+  }
+  .intro-text {
+    align-self: start;
+    width: 40%;
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css]: Same here

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 6]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
+  }
+}
+
+@media screen and (min-width: 900px) {
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css:265]: Where does this value of `900px` come from?

Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css]: Where does this value of 900px come from?

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 7]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
+  .card-section {
+    display: flex;
+    justify-content: center;
+    flex-flow: wrap;
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css:268]: What's the difference between `flex-wrap` and `flex-flow`?

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 8]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
+    gap: var(--gap);
+  }
+  .--forest {
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css:272]: Are you sure this follows BEM convention?

Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css:271]: Are you sure this follows BEM convention?

File: cssResponsive/css-responsive_responsive-website/css/styles.css [hunk 9]
```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
...
+    width: 86%;
```
Comment (reverbePrintemps) [inline cssResponsive/css-responsive_responsive-website/css/styles.css]: Where does this `86%` come from?

**Review by reverbePrintemps — APPROVED** [review]
Code looks relevant but you haven't provided any information in PR's description so I don't know what this work is about. A little context would've been helpful :) I'll approve the PR although I haven't witnessed your code live in the browser (you haven't provided information on how to run it locally) and I don't know where your [magic numbers](https://www.baeldung.com/cs/antipatterns-magic-numbers#magic-numbers) (ie. arbitrary values) come from.

File: cssResponsive/css-responsive_responsive-website/css/styles.css (hunk 7 above)
Comment (AntKarpo) [reply cssResponsive/css-responsive_responsive-website/css/styles.css:268]: so i dont need to do flex-direction but maybe i didnt need it in the first place. i will check it

Comment (AntKarpo) [reply cssResponsive/css-responsive_responsive-website/css/styles.css:268]: yeah so it doesnt work with flex-wrap: wrap; only
so flex-flow: wrap; makes more sence

File: cssResponsive/css-responsive_responsive-website/css/styles.css (hunk 1 above)
Comment (AntKarpo) [reply cssResponsive/css-responsive_responsive-website/css/styles.css]: yeah. i wasnt sure in the beginning if they wanted us to calculate it or not.
and later i forgot about it.

File: cssResponsive/css-responsive_responsive-website/css/styles.css (hunk 2 above)
Comment (AntKarpo) [reply cssResponsive/css-responsive_responsive-website/css/styles.css]: i used a width where i could see the changes the best (arbitrary)
600px and 900px was just so i had a better view of the changes
usually i would use 480 and 1100 for tablet and desktop

File: cssResponsive/css-responsive_responsive-website/css/styles.css (hunk 4 above)
Comment (AntKarpo) [reply cssResponsive/css-responsive_responsive-website/css/styles.css]: trial and error to see what fits best and looks closest to the preview

File: cssResponsive/css-responsive_responsive-website/css/styles.css (hunk 8 above)
Comment (AntKarpo) [reply cssResponsive/css-responsive_responsive-website/css/styles.css:271]: will look into it

**Pull request merged**